src/components/scene-components.js

```diff
@@ -29,3 +29,4 @@ import "./video-pause-state";
 import "./particle-emitter";
 import "./audio-zone";
 import "./audio-zone-source";
+import "./troika-text";
```

Comment:
The linter complaints about the missing extra blank line at the end of this file.